README.md

```diff
@@ -1,4 +1,6 @@
-[![Release](https://img.shields.io/github/release/redis/jedis.svg?sort=semver)](https://github.com/redis/jedis/releases/latest)
+# Jedis
+---
```

Comment:
Both `#` and `---` draws a line of their own. It becomes two back-to-back lines.
```suggestion
# Jedis

```